Moblin Watch/Various/Model.swift

```diff
@@ -246,6 +246,11 @@ class Model: NSObject, ObservableObject {
         if self.settings.show == nil {
             self.settings.show = .init()
         }
+        if self.settings.remoteControl == true {
+            previewTimeout = Duration.seconds(5)
+        } else {
+            previewTimeout = Duration.seconds(10)
+        }
```

Comment:
why is this needed?

Reply:
The interval for the clock is 5 seconds. For the remote assistant, it's 10 seconds. I have now also changed the interval for the remote assistant to 5 seconds. I hope that's okay.